src/test/groovy/graphql/execution/ValuesResolverTest.groovy

```diff
@@ -172,17 +172,20 @@ class ValuesResolverTest extends Specification {
         given: "schema defining input object"
         def inputObjectType = newInputObject()
                 .name("inputObject")
+                .field(newInputObjectField()
+                        .name("inputField")
+                        .type(GraphQLString))
                 .build()
 
-        def fieldArgument = newArgument().name("arg").type(inputObjectType).defaultValue("hello").build()
+        def fieldArgument = newArgument().name("arg").type(inputObjectType).defaultValueProgrammatic([inputField: "hello"]).build()
         def argument = new Argument("arg", new VariableReference("var"))
 
         when:
         def variables = CoercedVariables.emptyVariables()
         def values = ValuesResolver.getArgumentValues([fieldArgument], [argument], variables, graphQLContext, locale)
 
         then:
-        values['arg'] == 'hello'
+        values['arg'] == [inputField: 'hello']
```

Comment:
This is the only test in this PR that changed - a string result changed to a map.

This is why `defaultValue` is deprecated, it didn't pick up that the default value type was wrong! 😁